### PR TITLE
App listing custom elements

### PR DESCRIPTION
**Pull request by Simply007**
### Motivation

Create a website that would list custom elements with a short description to show the user what does this repository contains.

For review:
* Go through repository Readme if it does not miss some important info
* Go through application Readme if it is clear
* Try to run the marketplace locally and check the custom elements JSONs, if the information inside is correct

> `/marketplace` files are just a built marketplace site

### Checklist

- [x] Code follows coding conventions held in this repo
- [ ] Automated tests have been added
- [ ] Tests are passing
- [x] Docs have been updated (if applicable)
- [x] Temporary settings (e.g. variables used during development and testing) have been reverted to defaults

### CODE REVIEW

File: app/content/elements/GeolocationLeaflet.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "title": "Geoloation Leaflet",
```
Comment (makma) [inline app/content/elements/GeolocationLeaflet.json]:
```suggestion
  "title": "Geolocation Leaflet",
```

File: app/content/elements/IconSelector.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "title": "Icon Selector",
+  "description": "In Kentico Cloud, an icon selector can be specified by using the `Multiple Choice` content element. But this lacks in usablility as the content author has to guess what icon will be displayed when a particular choice is made. The icon selector hopes to make the life of the content editor easier with an easy to use dropdown of icons.",
```
Comment (makma) [inline app/content/elements/IconSelector.json]:
```suggestion
  "description": "In Kentico Cloud, an icon selector can be specified by using the `Multiple Choice` content element. But this lacks in usability as the content author has to guess what icon will be displayed when a particular choice is made. The icon selector hopes to make the life of the content editor easier with an easy to use dropdown of icons.",
```

File: app/content/elements/Magento.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+  "title": "Magento Product Selector",
+  "description": "Magento product selector is a selector connected to Magento instance which allows users to search and select a product from Magento product catalog.",
```
Comment (makma) [inline app/content/elements/Magento.json]:
```suggestion
  "description": "Magento product selector is a selector connected to Magento instance which allows users to search and select a product from the Magento product catalog.",
```

File: app/content/elements/Markdown.json [hunk 4]
```diff
@@ -0,0 +1,6 @@
+{
+  "title": "Markdown",
+  "description": "Markdown editor is a WYSIWYG Custom element which allows users to write formatted text using SimpleMDE. In this sample element, you can find examples of setting dynamic height and reacting on the window ‘resize’ events. When the element is disabled, the editor is set to the readonly mode.",
```
Comment (makma) [inline app/content/elements/Markdown.json]:
```suggestion
  "description": "Markdown editor is a WYSIWYG Custom element which allows users to write formatted text using SimpleMDE. In this sample element, you can find examples of setting dynamic height and reacting on the window ‘resize’ events. When the element is disabled, the editor is set to the read-only mode.",
```

File: app/content/elements/Optimizely.json [hunk 5]
```diff
@@ -0,0 +1,6 @@
+{
+  "title": "Optimizely Audience selector",
+  "description": "Optimizely audiences selector is a selector connected to Optimizely API which allows users to search and select audiences from Optimizely.",
```
Comment (makma) [inline app/content/elements/Optimizely.json]:
```suggestion
  "description": "Optimizely audiences selector is a selector connected to Optimizely API which allows users to search and select audiences from the Optimizely.",
```

File: app/content/elements/YoutubeVideoPreview.json [hunk 6]
```diff
@@ -0,0 +1,6 @@
+{
+  "title": "Youtube Video Preview",
+  "description": "This KC custom element allows you to add various types of Youtube video url, it extracts the video id for your apps and shows the video.",
```
Comment (makma) [inline app/content/elements/YoutubeVideoPreview.json]:
```suggestion
  "description": "This KC custom element allows you to add various types of Youtube video URL, it extracts the video id for your apps and shows the video.",
```

File: app/README.md [hunk 7]
```diff
@@ -0,0 +1,81 @@
+# Custom Elements Marketplace
```
[the remark this answers is not in the document]
Comment (Simply007) [reply app/README.md]: I have chosen `marketplace-src`

**Pull request merged**